python/ray/reporter.py

```diff
@@ -62,6 +61,7 @@ def GetProfilingStats(self, request, context):
     def ReportMetrics(self, request, context):
         # NOTE: Exceptions are not propagated properly
         # when we don't catch them here.
+        print(request)
```

Comment:
keep?

Reply:
Sorry. I totally forgot about it. Will remove.